### PR TITLE
Pom cleanup: Added values for name,descrption,url tag

### PR DESCRIPTION
**Pull request by anantk24**
- added appropriate naming conventions for name tag

- added appropriate description for each module

- added parent url for url tag

### CODE REVIEW

**Comment by karianna** [comment]
@anantk24 Apologies but a major refactor went in so you may need to rabase or just restart this one :-(

**Comment by anantk24** [comment]
> @anantk24 Apologies but a major refactor went in so you may need to rabase or just restart this one :-(



> @anantk24 Apologies but a major refactor went in so you may need to rabase or just restart this one :-(

Yes. No issues, will update it.

File: vertx/pom.xml [hunk 1]
```diff
@@ -10,7 +10,9 @@
     </parent>
 
     <artifactId>gctoolkit-vertx</artifactId>
-
+    <name>GCToolKit Vertx</name>
+    <description>Event module for creating discrete event from parsing log files.</description>
```
Comment (dsgrieve) [inline vertx/pom.xml:14]: Messaging backplane for connecting GC log parsers and JVM event consumers.

Comment (dsgrieve) [reply vertx/pom.xml:14]: Just to be clear, there may be more than one parser active. A line is read from a GC log and published on the message bus, and then the parsers publish JVMEvent's on the message bus (different channels than the log line, of course!) where the events are picked up by the Aggregator's.

**Review by brunoborges — APPROVED** [review]
I've made commits to @anantk24 's branch, and also applied @dsgrieve 's suggestions. All seems good now.

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**